server/historian/packages/historian-base/package.json

```diff
@@ -26,10 +26,30 @@
 		"prettier:fix": "prettier --write . --ignore-path ../../.prettierignore",
 		"start": "node dist/www.js",
 		"test": "mocha --recursive dist/test",
-		"test:coverage": "nyc npm test -- --reporter xunit --reporter-option output=nyc/junit-report.xml",
+		"test:coverage": "c8 npm test -- --reporter xunit --reporter-option output=nyc/junit-report.xml",
```

Comment:
Any reason you started preserving '--reporter xunit ...' at this point?

(Previously you had been removing it.)

Reply:
With PR #17173 and #17145 and #17082, I have made client release group to always generate report whether we are in CI or not via config. So any extra report flag is not necessary.

server release groups hasn't switched to that yet.  So I am preserving those for now.